jira/manifest.yml

```diff
@@ -0,0 +1,23 @@
+modules:
+  trigger:
+    - key: view-activity
+      function: view
+      events:
+        - avi:jira:viewed:issue
+  function:
+    - key: view
+      handler: index.eventHandler
+app:
+  id: ari:cloud:ecosystem::app/99fb9271-5707-4cce-9f9d-6c65a37e607c
+permissions:
+  scopes:
+    - read:issue:jira
+    - read:issue-type:jira
+    - read:user:jira
+    - read:project:jira
+    - read:status:jira
+    - read:jira-work
+  external:
+    fetch:
+      backend:
```

Comment:
Need to replace this with `scio-apps` central_datasources `JiraForgeEventHandler` URL. For external permissions: [link](https://developer.atlassian.com/platform/forge/manifest-reference/permissions/#external-permissions)